l10n_br_fiscal/models/document_fiscal_line_mixin.py

```diff
@@ -424,6 +424,12 @@ def _operation_domain(self):
         string="Tax ICMS FCP",
         domain=[('tax_domain', '=', TAX_DOMAIN_ICMS_FCP)])
 
+    # vBCFCPUFDest
+    icmsfcp_base = fields.Monetary(
+        string="ICMS FCP Base",
+        default=0.00,
```

Comment:
tira esse default=0.00, serve para nada e veja o impacto deles aqui https://github.com/OCA/l10n-brazil/pull/1107

Reply:
Rebased and removed attribute default=0.00